_posts/2021-10-25-g2p-applications.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title:  "Applications for g2p, an example with ReadAlongs"
+author: AidanPine
+categories: []
+tags: [ tutorial, explanation, tech-showcase, g2p ]
+image: assets/images/nikikw.png
+description: "Applications for g2p, an example with ReadAlongs"
+featured: false
+hidden: false
+---
+
+This is the 5th post in a seven-part series about g2p. In this post, we discuss using g2p to create interactive text/audio books with ReadAlong Studio.
+
+### G2P Blog Series Index
+
+- [Background]({{ "g2p-background" | absolute_url }})
+- [How to write a basic mapping in G2P Studio]({{ "g2p-basic-mappings-gui" | absolute_url }})
+- [Writing mappings on your computer]({{ "g2p-basic-mappings-local" | absolute_url }})
+- [Advanced mappings]({{ "g2p-advanced-mappings" | absolute_url }})
+- ReadAlong Studio & Other Applications
+- [Preprocessing mappings]({{ "g2p-preprocess" | absolute_url }})
+- [Contributing]({{ "g2p-contributing" | absolute_url }})
+
+This post will discuss the use of `g2p` in the [ReadAlongs](#readalongs) project for creating interactive audio/text documents.
+
+## Other Applications
+
+There are number of different software tools that are already making use of `g2p`. For general purpose use in Python, have a look at [this post]({{ "g2p-advanced-mappings" | absolute_url }}).
+
+### Convertextract
+
+Convertextract is a tool that lets you use `g2p` mappings to convert text inside a variety of different documents, including Microsoft Office documents like Powerpoint and Word coduments. Have a look at the [convertextract repo readme](https://github.com/roedoejet/convertextract) or at Fineen Davis' [blog post]({{ "convertextract-app" | absolute_url }}) about the Convertextract GUI.
+
+### ReadAlongs
+
+ReadAlongs is a research project from the [National Research Council's Indigenous Language Technology Project](https://nrc.canada.ca/en/research-development/research-collaboration/programs/canadian-indigenous-languages-technology-project).
+
+Language revitalization communities have potentially many different recordings of language and some associated text, but mobilizing these materials into something that is educationally useful can be time consuming. It's also tough from the learner's perspective to try and follow along the text (maybe a Word Document) just by simply playing the audio. It's easy to get lost in the recording quickly, and from personal experience, trying to do that usually involves a lot of frustrating rewinding.
+
+What if we could develop a tool that *automatically* figured out what parts of the audio file corresponded to what parts of the text? In [Natural Language Processing](https://en.wikipedia.org/wiki/Natural_language_processing), this is called *forced alignment*, and this is fundamentally what the ReadAlongs project does. But how does it do it? In part, by using a whole bunch of `g2p` mappings!
```

Comment:
Hum, I think you should have a hyperlink to the readalongs project itself between the title of this section and this paragraph. Maybe on "ReadAlongs project" right in this paragraph.
This link could work: https://nrc.canada.ca/en/research-development/research-collaboration/programs/readalong-studio-application-indigenous-audiobooks-videos-project
it has outgoing links to GitHub and other relevant things.

Reply:
Or a link to Studio's github page would work too, since it also has a link back to the canada.ca page. It's all connected. :)